spec/unit/rust-crypto/rust-crypto.spec.ts

```diff
@@ -193,6 +193,10 @@ describe("initRustCrypto", () => {
 
             fetchMock.get("path:/_matrix/client/v3/room_keys/version", { version: "45" });
 
+            function legacyMigrationProgressListener(progress: number, total: number): void {
+                logger.log(`migrated ${progress} of ${total}`);
```

Comment:
Should we be logging this in a unit test?

Reply:
I figured it was fairly harmless, and possibly useful when debugging the test. As far as I know there is no particular embargo against logging in tests?

Reply:
I find unit test output hard to understand because of all the logging. My understanding was/is that well-behaved tests would avoid logging, and we have `filterConsole` in matrix-react-sdk as an attempt to document expected console messages without filling the console with spam.

Or is use of `logger` better than `console` in some way? Maybe it doesn't fill the console?

My expectation might be wrong anyway. I just don't think I've seen tests deliberately logging before and wanted to check you meant it.

Reply:
`logger` is no better than `console` as far as I know.

You may be right that the amount of spam from tests is confusing; however this is far from the only thing that gets written.

I kinda wish that jest had better facilities for handling console output from tests. For example, it might default to only showing the log output from failing tests.

Reply:
Agreed.